css/styles.css

```diff
@@ -0,0 +1,839 @@
+  }
+
+  nav ul li {
+    margin-left: 15px;
+    margin-right: 15px;
+    position: relative;
+  }
+
+  nav ul li a {
+    font-weight: 600;
+    color: #172b4d;
+  }
+}
+
+h1 {
+  padding-top: 20px;
+  font-size: 40px;
+  text-align: center;
+  color: #172b4d;
+  font-weight: 700;
+  line-height: 52px;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+h1 span {
+  display: block;
+}
+
+h1 span:last-child {
+  color: #36b37f;
+}
+
+h5 {
+  color: #172b4d;
+  font-size: 20px;
+  line-height: 28px;
+  padding-left: 25px;
+  padding-right: 25px;
+  padding-top: 5px;
+  text-align: center;
+  font-weight: 400;
+}
+
+.social {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  list-style: none;
+  padding: 35px 25px 20px 25px;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
+.social li {
+  margin-left: 10px;
+  margin-right: 10px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.arrow-wrapper img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  transition: all 0.5s;
+}
+
+.social li img {
+  display: block;
+}
+
+.arrow-wrapper img:hover {
+  transform: translateY(8px);
+  transition: all 0.5s;
+}
+
+.banner-section {
+  text-align: center;
+  max-width: 710px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.banner-section p {
+  padding-left: 24px;
+  padding-right: 24px;
+  font-size: 20px;
+  line-height: 28px;
+  letter-spacing: -0.37px;
+}
+
+@media (min-width: 768px) {
+  .banner-section p {
+    font-size: 18px;
+  }
+
+  h1 {
+    padding-top: 12%;
+  }
+}
+
+.arrow-wrapper {
+  padding-top: 30px;
+  display: inline-block;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.headlined {
+  display: flex;
+  flex-direction: column;
+
+  /* border: 2px solid red; */
+  width: 920px;
+  height: 756px;
+  top: 72px;
+  margin: auto;
+}
+
+.headlinecontents {
+  height: 114px;
+  width: 920px;
+  text-align: center;
+}
+
+.headlinecontents h4 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 48px;
+  line-height: 60px;
+  letter-spacing: 0.4px;
+  color: #172b4d;
+}
+
+.green {
+  color: green;
+}
+
+.text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #172b4d;
+}
+
+.ul-socialmedia {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: auto;
+  width: 200px;
+  height: 40px;
+  margin-top: 28px;
+  padding-inline-start: 0;
+}
+
+.socialists {
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.sec {
+  width: 40px;
+  height: 40px;
+  margin-top: 30px;
+  border-radius: 32px;
+  border: 1px solid #dfe1e6;
+}
+
+.down {
+  width: 14px;
+  height: 8px;
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  padding: 80px, 0, 80px, 0;
+  gap: 48px;
```

Comment:
- [ ] There shouldn't be any commas here, it should be like this:  `padding: 80px 0 80px 0;` kindly fix it. 😁

Reply:
Resolved